tests/testsuite/check.rs

```diff
@@ -347,7 +347,8 @@ fn rustc_check_err() {
         .with_status(101)
         .with_stderr_contains("[CHECKING] bar [..]")
         .with_stderr_contains("[CHECKING] foo [..]")
-        .with_stderr_contains("[..]cannot find function `qux` in [..] `bar`")
+        .with_stderr_contains("[..]cannot find function `qux`[..]")
```

Comment:
Should we stop asserting the string, and instead check only the appearance of `E0425`? I feel like the error code will always be there, no?

Reply:
The error code will indeed always be there. I'm ok with either approach.

Reply:
@epage
Any opinion on adding a redaction for the first line rustc diagnostic message? Something like

```
error[E0425]: [RUSTC_ERROR_MESSAGE]
```

We still need to use `...` to omit the multiline sub-diagnostics though.

Reply:
Interesting, not a bad idea, but you can already accomplish that with `error[E025]: [..]` right?

Reply:
So we only match on the error number and not the error message?  I guess that would be specific enough.

Reply:
> Interesting, not a bad idea, but you can already accomplish that with error[E025]: [..] right?

The more we can auto-redact, the easier to is to write new tests and do the right thing (or update existing tests)